test/CAS/Tests/AuthenticationTest.php

```diff
@@ -68,7 +72,7 @@ protected function setUp()
         $_SERVER['PHP_SELF'] = '/index.php';
         $_SESSION = array();
 
-        $this->object = new CAS_Client(
+        $this->object = new \CAS_Client(
```

Comment:
I am thinking if `use CAS_client;` and eliminating `\` before `CAS_Client` would be better (can be applied on `\PhpCas::` as well), but this is more of a coding style question and can be addressed later when we move everything to PSR-4.

Reply:
> can be applied on \PhpCas:: as well

Right now, there are no cases of `PhpCas` with a leading backslash.

If you'd like me to do this for `CAS_*` classes as well, just let me know. But it seems you may be on the fence about it? I'm good either way.

Reply:
One of the `PhpCas::` occurrences: https://github.com/apereo/phpCAS/pull/332/files#diff-97c147f192af785e25d0a34a0ac8e71cR42

I think this can be left to be polished later though.